src/Evolution/Initialization/DgDomain.hpp

```diff
@@ -142,6 +143,14 @@ struct Domain {
         element_id, my_block.is_time_dependent()
                         ? my_block.moving_mesh_logical_to_grid_map().get_clone()
                         : my_block.stationary_map().get_to_grid_frame()};
+    ElementMap<Dim, Frame::Inertial> element_map_inertial{
+        element_id, my_block.is_time_dependent()
+                        // In case of a time dependent domain, this
+                        // element map should / would not be used
```

Comment:
I don't understand this comment...can you please clarify? Does this code work correctly with moving mesh and time-dependent maps? Does it work with stationary maps as well? Or both?

Reply:
removed commit